rebar.config

```diff
@@ -9,7 +9,7 @@
 {deps,
  [
   lager,
-  {cuttlefish, ".*", {git, "https://github.com/tsloughter/cuttlefish.git", {branch, "develop"}}},
+  {cuttlefish, "2.0.7"},
```

Comment:
this is not needed any more, it shouldn't be included or it can conflict with the cuttlefish plugin